### PR TITLE
updated documentation build steps with pkg-config note

### PR DESCRIPTION
**Pull request by KozzyKoder**
Build process requires pkg-config to be installed. Since it is not obvious step makes sense to include its installation steps into documentation.

### CODE REVIEW

**Comment by grzkv** [comment]
@gksinghjsr Did you encounter any of the problems described in this addition?

**Comment by KozzyKoder** [comment]
Yep, failed to install 2 times:
first time it failed because of glib dependency. Key --with-internal-glib solved this issue for me.
Next time it failed with this flag, since this flag requires env variable LDFLAGS to be set up.

File: README.md [hunk 1]
```diff
@@ -31,6 +31,20 @@ To build the binaries with debug symbols, run:
 make debug
 ```
 
+Note: build process requires pkg-config to be installed:
+
+### Mac OS X
+
+```
+curl https://pkgconfig.freedesktop.org/releases/pkg-config-0.29.tar.gz -o pkgconfig.tgz
```
Comment (grzkv) [inline README.md]: These version numbers will not last.

**Comment by grzkv** [comment]
How about removing specific version numbers? These will become obsolete soon.

**Comment by grzkv** [comment]
I guess this should be described as a special case. I did not encounter problems like this when doing that on MacOS. How about you @gksinghjsr ?

File: README.md (hunk 1 above)
Comment (KozzyKoder) [reply README.md]: removed specific version of pkg-config. User can specify version that fits him the best.

**Comment by grzkv** [comment]
LGTM! Thanks for the pull-request!

**Pull request merged**